scripts/lib/mkcloud-common.sh

```diff
@@ -1,6 +1,12 @@
 # This file shares common code between mkcloud-${mkclouddriver}.sh files and qa_crowbarsetup.sh
 
 # ---- START: functions related to repos and distribution settings
+# default vlans in crowbar
+vlan_storage=${vlan_storage:-200}
+vlan_ceph=${vlan_ceph:-600}
+vlan_public=${vlan_public:-300}
+vlan_fixed=${vlan_fixed:-500}
+vlan_sdn=${vlan_sdn:-400}
```

Comment:
I am puzzled why this would fix this problem. This file is source in `mkcloud` already in line 60. So why was `vlan_public` not set later on?

Reply:
Do you have a recent Jenkins log with that error?

Reply:
the defaulting previously happened in setcloudnetvars which is not always called (because it will not work without a cloud name)

Reply:
True, good point.